CMakeLists.txt

```diff
@@ -21,15 +21,18 @@ endif()
 
 add_subdirectory(third_party/pybind11)
 
+################################
+# Core
+################################
 set(TENSEAL_BASEDIR tenseal)
 set(SOURCES
-    ${TENSEAL_BASEDIR}/binding.cpp
     ${TENSEAL_BASEDIR}/tensealcontext.cpp
     ${TENSEAL_BASEDIR}/tensors/bfvvector.cpp
     ${TENSEAL_BASEDIR}/tensors/ckksvector.cpp
     ${TENSEAL_BASEDIR}/utils/utils.cpp
 )
-pybind11_add_module(_tenseal_cpp ${SOURCES})
+add_library(libtenseal SHARED ${SOURCES})
```

Comment:
This could be conditioned by an environment variable BUILD_TEST

Reply:
It doesn't affect the actual library